letsdrill.rb

```diff
@@ -1,12 +1,28 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ 	if integer > 89
+ 		return "A"
+ 	elsif integer > 79
+ 		return "B"
+ 	elsif integer > 69
+ 		return "C"
+ 	elsif integer > 59
+ 		return "D"
+ 	end
+ 	return "F"
 
 end
 
 def shortest_string(array)
-
-  #Put your code here!
+	if array == []
+		return nil
+	end
+	short = array[0] 
+	array.each do |str|
+		if str.length < short.length
+			short = str
+		end
+	end
+	return short
```

Comment:
Would you believe Ruby has a method builtin that does all this? Check out [`Enumerable#min_by`](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-min_by). (`Enumerable` is included in `Array`.) Scan through those Enumerable methods, there's a bunch of neat stuff in there.